README.md

```diff
@@ -74,8 +73,8 @@ You need to download the mentioned databases and specify them as inputs to param
 ## Pipeline inputs
 
 > [!NOTE]
-> If you want to use the pipeline on [ENA](https://www.ebi.ac.uk/ena/browser/home), data follow these [instructions](docs/ena_readme.md). 
-> Otherwise, download your data and organise it in the recommended format described below.
+> If you own your studies of interest, they have been deposited to [ENA](https://www.ebi.ac.uk/ena/browser/home) and you own them, follow these [instructions](docs/ena_readme.md).
```

Comment:
I'm not sure that word "own" is clear here. It is clear for me, but maybe not for outside user...